.rubocop.yml

```diff
@@ -136,6 +136,10 @@ Rails/Output:
     - 'lib/audit/catalog_to_moab.rb' # use puts statements for following progress of long job
     - 'lib/audit/moab_to_catalog.rb' # use puts statements for following progress of long job
 
+Rails/TimeZone:
+  Exclude:
+    - 'app/models/preserved_copy.rb' # use Time.parse without zone
+
 # --- RSpec ---
```

Comment:
I can get all specs/cops to pass without this exception by following one of Rubocop's suggestions -- see next comment.